### PR TITLE
docs(input): remove extra parens and fix nits

### PR DESCRIPTION
**Pull request by willshowell**

### CODE REVIEW

File: src/lib/input/input.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ elements inside `md-form-field` as well. This includes Angular directives such a
 
 The only limitations are that the `type` attribute can only be one of the values supported by
 `mdInput` and the native element cannot specify a `placeholder` attribute if the `md-form-field`
-also contains a `md-placeholder` element.
+also contains an `md-placeholder` element.
```
Comment (rafaelss95) [inline src/lib/input/input.md]: `a` was correct, no?

File: src/lib/input/input.md [hunk 2]
```diff
@@ -129,18 +129,18 @@ the error messages.
 function myErrorStateMatcher(control: FormControl, form: FormGroupDirective | NgForm): boolean {
   // Error when invalid control is dirty, touched, or submitted
   const isSubmitted = form && form.submitted;
-  return !!(control.invalid && (control.dirty || control.touched || isSubmitted)));
+  return !!(control.invalid && (control.dirty || control.touched || isSubmitted));
```
Comment (rafaelss95) [inline src/lib/input/input.md]: The _coercion_ here seems to be unnecessary since all fields are boolean.

File: src/lib/input/input.md (hunk 1 above)
Comment (willshowell) [inline src/lib/input/input.md]: Arguably haha. I'm basing it on the pronunciation of "md" sounding like "em dee". See [APA blog](http://blog.apastyle.org/apastyle/2012/04/using-a-or-an-with-acronyms-and-abbreviations.html) referencing the subject.

File: src/lib/input/input.md (hunk 2 above)
Comment (willshowell) [inline src/lib/input/input.md]: Yeah it is unnecessary, but it's based on the default errorStateMatcher which originates from https://github.com/angular/material2/pull/3560#discussion_r106483976.

Happy to change it if coercion is unnecessary, but it does add a little type safety. A little more discussion in https://github.com/angular/material2/issues/6585

Comment (jelbourn) [inline src/lib/input/input.md]: It has an effect because `isSubmitted` is potentially `undefined` (if `form` is `undefined`)

**Pull request merged**